neonwatty/meme_search
# docker-compose.yml improvements

## Changes made

- Rename compose.yaml to docker-compose.yml
- Formatted the docker-compose.yml to [composeV2](https://docs.docker.com/compose/migrate/)
- Changed the ports to match 8555 -> 8501
- Remove unnecessary data volume definition
- Ability to use Nvidia GPU inside Docker Container

## How to test

- Clone repository
- Run `docker compose up` 
- Navigate to http://localhost:8501
- Check if the app behaves as intended

Let me know if I can contribute more to the Docker side of things.
